vendor/engines/bulk_email/config/locales/en.yml

```diff
@@ -11,4 +11,20 @@ en:
       start: Starting
       end: Ending
     no_results: No users found
-    export_html: Export
+    compose_mail: Compose Mail
+    export: Export
+    body:
+      greeting: Hello %{recipient_name},
+      signoff: If you have any questions, please contact %{facility_name}.
```

Comment:
And I'm thinking we should probably say something about being sent on behalf of `!app_name!`. We can punt on that, though.